src/sentry/plugins/bases/issue2.py

```diff
@@ -215,20 +249,27 @@ def view_create(self, request, group, **kwargs):
             return Response({'error_type': 'validation', 'errors': errors}, status=400)
 
         try:
-            issue_id = self.create_issue(
+            issue = self.create_issue(
```

Comment:
actually this likely still isnt legacy compat fully

im going to get some tests in to ensure it doesnt break existing behavior

Reply:
this comment I believe is addressed, as we change 'issue' below

that said, there's no tests proving this works or doesnt work